zookeeper-server/src/main/java/org/apache/zookeeper/ZooDefs.java

```diff
@@ -109,45 +111,43 @@
         int ADMIN = 1 << 4;
 
         int ALL = READ | WRITE | CREATE | DELETE | ADMIN;
+
     }
 
     @InterfaceAudience.Public
     public interface Ids {
+
         /**
          * This Id represents anyone.
          */
-        public final Id ANYONE_ID_UNSAFE = new Id("world", "anyone");
+        Id ANYONE_ID_UNSAFE = new Id("world", "anyone");
```

Comment:
this removed `final`.. intended?

Reply:
Any field in an interface must be `public static final`. There is a rule `redundant modifier` check this.